src/sssom/parsers.py

```diff
@@ -247,22 +250,35 @@ def parse_sssom_rdf(
 
 
 def parse_sssom_json(
-    file_path: str,
-    prefix_map: ConverterHint = None,
-    meta: Optional[MetadataType] = None,
-    **kwargs
-    # mapping_predicates: Optional[List[str]] = None,
+    file_path: str, prefix_map: ConverterHint = None, meta: Optional[MetadataType] = None, **kwargs
 ) -> MappingSetDataFrame:
-    """Parse a TSV to a :class:`MappingSetDocument` to a  :class`MappingSetDataFrame`."""
+    """Parse a TSV to a :class:`MappingSetDocument` to a :class:`MappingSetDataFrame`."""
     raise_for_bad_path(file_path)
-    converter, meta = _get_prefix_map_and_metadata(prefix_map=prefix_map, meta=meta)
 
     with open(file_path) as json_file:
         jsondoc = json.load(json_file)
+    context_from_jsondoc = jsondoc.get(JSON_CONTEXT_KEY)
+
+    # Initialize meta if it's None
+    if meta is None:
+        meta = {}
+
+    # Update metadata with values from JSON document
+    meta_keys_to_update = [MAPPING_SET_ID, LICENSE]
+    meta.update({key: jsondoc[key] for key in meta_keys_to_update if key in jsondoc})
+
+    # Update CURIE_MAP if context_from_jsondoc exists
+    if context_from_jsondoc:
+        prefixes_from_json = {
+            key: value
+            for key, value in context_from_jsondoc.items()
+            if isinstance(value, str) and not key.startswith("@")
+        }
```

Comment:
Could the above be shortened by using `curies.Converter.from_json().bimap`?

https://curies.readthedocs.io/en/latest/api/curies.Converter.html#curies.Converter.from_jsonld

Reply:
You're absolutely right! Great catch, thanks!